### PR TITLE
Links between Variables and WindowRules for renderunfocused

### PR DESCRIPTION
**Pull request by Hannes1909**
also some minor linting in Variables.md and Window-Rules.md
https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md055.md
https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md031.md

Added sh as language for IconList, so linter wouldnt throw a warning.

### CODE REVIEW

File: pages/Configuring/Window-Rules.md [hunk 1]
```diff
@@ -316,7 +314,7 @@ where `rule` is the rule and `namespace` is the namespace regex (find
 namespaces in `hyprctl layers`) or `address` is an address in the form of
 `address:0x[hex]`.
 
-### Rules
+### List of Rules
```
Comment (fufexan) [inline pages/Configuring/Window-Rules.md:317]: This will break existing links. I'd rather leave headings as they are.

Comment (Hannes1909) [reply pages/Configuring/Window-Rules.md:317]: the linter complained that there is already a heading named "rules", i couldnt find anything that linked here, just at ##rules in line 95. but i can revert that change and just commit the others, if thats ok?

Comment (fufexan) [reply pages/Configuring/Window-Rules.md:317]: Yeah, the rest are ok.

Comment (Hannes1909) [reply pages/Configuring/Window-Rules.md]: Newest commit should be good to go :)
can we merge it like that or do i have to open another PR - havent contributed much in public projects up until now, sorry if thats a newb question.

Comment (fufexan) [reply pages/Configuring/Window-Rules.md]: Yeah, it can be merged. No need for another PR.

**Review by fufexan — APPROVED** [review]

**Pull request merged**